eth-withdrawal/src/main/kotlin/com/d3/eth/withdrawal/withdrawalservice/WithdrawalServiceInitialization.kt

```diff
@@ -51,26 +101,32 @@ class WithdrawalServiceInitialization(
         logger.info { "Init Iroha chain listener" }
         return chainListener.getBlockObservable()
             .map { observable ->
-                observable.flatMapIterable { (block, _) -> IrohaChainHandler().parseBlock(block) }
+                observable.flatMapIterable { (block, _) ->
+                    ethereumWithdrawalExpansionStrategy.filterAndExpand(block)
+                    IrohaChainHandler().parseBlock(block)
```

Comment:
it is already done in `filterAndExpand(block)`, isn't it?

Reply:
No, ethereumWithdrawalExpansionStrategy.filterAndExpand(block) responsible only for expansion, the rest is performed by parseBlock()

Reply:
Why do you have `IrohaChainHandler().parseBlock(block)` inside `filterAndExpand()`
EthereumWithdrawalExpansionStrategy:51

Reply:
fixed